3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,131 @@
+
+let yearOfBirth = 1982;
+let yearFuture = 2022;
+let age = yearFuture - yearOfBirth;
+console.log(`you will be ${age} years old in ${yearFuture}`);
+
+//Dog year of birth
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let humanYears = dogYearFuture - dogYearOfBirth; 
+let dogYear = humanYears*7; 
+let shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears) {
+let dogAgeInDogYear = dogYear;
+
+console.log(`Your dog will be ${dogYear} dog years in ${dogYearFuture}`);
+}
+else {
+    console.log(`Your dog will be ${humanYears} human years old in ${dogYearFuture}`);
+}
+
```

Comment:
🐶 Your distinction between humanYears and dogYear is smart and readable.
💡 Similarly, since dogYearOfBirth, dogYearFuture, and shouldShowResultInDogYears aren't reassigned, they can be declared with const.
🔧 A small note: you've declared let dogAgeInDogYear = dogYear; inside the if block but don't seem to use dogAgeInDogYear. Consider removing this line for cleaner code.

Reply:
Please recheck my editted code in this block.